### ISSUE TITLE
Dash-compatible docs

### ISSUE THREAD

**Issue by noahsark769** [noahsark769/cifilter.io#2]
https://kapeli.com/docsets

**Comment by jessesquires** [comment]
Not 100% sure what you mean by this, but you can use [Jazzy](https://github.com/realm/jazzy) which generates docsets.

**Comment by noahsark769** [comment]
Yup! I think with dash you can export any html as a docset, but not completely sure. I was thinking a docset for each different page on CIFilter.io, although jazzy could be another option 👍
